platforms/jvm/jvm-services/build.gradle.kts

```diff
@@ -59,3 +59,9 @@ dependencies {
 
     integTestDistributionRuntimeOnly(projects.distributionsCore)
 }
+
+packageCycles {
+    // Needed for the factory methods in the interface
```

Comment:
💭 Can we provide a better explanation for why this is necessary, and why it's ok (rather than resolving the cycle in some way)?

Reply:
I have clarified this. Let me know if that's not good enough. I also made the exclusion more focused.